pkgs/top-level/python-packages.nix

```diff
@@ -18,7 +18,7 @@ let
   packages = ( self:
 
 let
-  inherit (python.passthru) isPy27 isPy33 isPy34 isPy35 isPy36 isPy37 isPy38 isPy3k isPyPy pythonAtLeast pythonOlder;
+  inherit (python.passthru) isPy27 isPy2 isPy33 isPy34 isPy35 isPy36 isPy37 isPy38 isPy3k isPyPy pythonAtLeast pythonOlder;
```

Comment:
@fridh this used to be exported, and is still set on python itself, was isPy2 removed from a reason?

Reply:
yes it was removed because `!isPy3k` achieves the same and because the naming was inconsistent with `isPy3k`.

Reply:
Hmm.
- `!isPy3k` is technically wrong when we want to express that it only works in Python 2.x and will break when Python 4 is released
- `Py3k` is not wrongly named, it [was](https://www.python.org/dev/peps/pep-3000/#naming)/is a name used for [Python 3](https://docs.python.org/3/whatsnew/3.0.html)

But yeah, if we've migrated away from it already, I'll do that here, too.